addons/theme_default/views/theme_default_templates.xml

```diff
@@ -18,107 +18,41 @@
-                              </label>
-                          </td>
-                          <td>
-                              <label class="chd-color-combi">
-                                  <img src="/theme_default/static/src/img/theme/variant-ruby.gif" alt="Blue" class="chd-color-combi-img"/>
-                                  <input name="colorvar" type="radio" data-xmlid="theme_default.option_color_ruby_variables,theme_default.option_color_ruby_bs_variables,theme_default.option_color_ruby"/>
-                              </label>
-                          </td>
-                          <td>
-                              <label class="chd-color-combi">
-                                  <img src="/theme_default/static/src/img/theme/variant-gold.gif" alt="Gold" class="chd-color-combi-img"/>
-                                  <input name="colorvar" type="radio" data-xmlid="theme_default.option_color_gold_variables,theme_default.option_color_gold_bs_variables,theme_default.option_color_gold"/>
-                              </label>
-                          </td>
-                      </tr>
-                  </table>
-
-                  <h5 class="modal-h5">FONTS COMBINATIONS</h5>
-                  <table name="font">
-                      <tr>
-                          <td width="50%">
-                              <label>
-                                  <div>
-                                      <span style="font-family:'Helvetica Neue', Helvetica, Arial, sans-serif; font-size:11px">Helvetica</span>
-                                      <span style="font-family:Georgia, 'Times New Roman', Times, serif; font-size:11px" >/ Georgia</span>
-                                  </div>
-                                  <input name="theme" type="radio" data-xmlid=""/>
-                              </label>
-                          </td>
-                          <td width="50%">
-                              <label>
-                                  <div>
-                                      <span style="font-family:Georgia, 'Times New Roman', Times, serif; font-size:11px" >Georgia</span>
-                                      <span style="font-family:'Helvetica Neue', Helvetica, Arial, sans-serif; font-size:11px">/ Helvetica</span>
-                                    <input name="theme" type="radio" data-xmlid="theme_default.option_font"/>
-                                  </div>
-                              </label>
-                          </td>
-                      </tr>
-                  </table>
-              </main>
-          </div>
-      </div>
-  </div>
-  </xpath>
+    <xpath expr="div" position="replace">
+        <div role="dialog" id="theme_customize_modal" class="modal fade">
+            <content data-string="Format" data-id="format" data-class="o_multi_choice active" data-classactive="active" data-title="Choose your layout" data-listclass="o_multichoice_list">
```

Comment:
`data-listclass` seems unused and I don't think it will be needed. Avoid data- attributes which are not *really* needed, the less code, the better. data-classactive seems useless too.